server/app.ts

```diff
@@ -14,7 +14,12 @@ app.use(express.static(`${__dirname}/public`));
 
 app.use(cors({
   credentials: true,
-  exposedHeaders: ['set-cookie'],
+  origin: true,
+}));
+
+app.options('*', () => cors({
+  credentials: true,
+  origin: true,
```

Comment:
setting `origin` in 17 and 22 lines it's not the same thing?

Reply:
No it's not. 22 is setting cors for all OPTIONS request however line 17 sets cors just for simple POST and GET. We need it to be set also to OPTIONS because of our request (checked via morgan on BE)